manifests/client.pp

```diff
@@ -214,6 +218,7 @@
   $shared_ca            = undef,
   $custom_options       = {},
   $expire               = undef,
+  $reneg_sec            = '',
```

Comment:
This does not match the documentation and IMHO a default value of `undef` is more consistent than an empty string. Could you change this?